.github/workflows/rails_tests.yml

```diff
@@ -18,10 +18,31 @@ jobs:
     outputs:
       matrix: ${{ steps.set-matrix.outputs.matrix }}
     steps:
+      - name: Checkout
+        uses: actions/checkout@11bd71901bbe5b1630ceea73d27597364c9af683 # v4
```

Comment:
we need checkout to support automated generation of the matrix from rspec_buckets.json